component/src/ibc/component.rs

```diff
@@ -31,7 +34,12 @@ impl IBCComponent {
     pub async fn new(state: State) -> Self {
         let client = Ics2Client::new(state.clone()).await;
         let connection = connection::ConnectionComponent::new(state.clone()).await;
-        let channel = channel::ICS4Channel::new(state.clone()).await;
+
+        let mut router = AppRouter::new();
+        let transfer = ICS20Transfer::new(state.clone());
+        router.bind(PortId::transfer(), Box::new(transfer));
```

Comment:
for future IBC apps, we would do 

`router.bind(PortId::interchain_accounts(), Box::new(interchain_acct_impl))`